src/query/frontend/opencypher/grammar/MemgraphCypher.g4

```diff
@@ -293,6 +296,8 @@ showUsersForRole : SHOW USERS FOR role=userOrRoleName ;
 
 dumpQuery: DUMP DATABASE ;
 
+analyzeGraphQuery: ANALYZE GRAPH ( ON LABELS ( listOfColonSymbolicNames | ASTERISK ) ) ? ( DELETE STATISTICS ) ? ;
```

Comment:
what happens if I specify

ANALYZE GRAPH ON LABELS * DELETE STATISTICS? 

I think it's a mutually exclusive case, should be separated with a |

Reply:
> what happens if I specify:
It works okay.

Why is exclusive? The user can say which labels to delete

Reply:
I think this is correct how it is.